provisioning/local/gcp_serviceAccount.tf

```diff
@@ -0,0 +1,34 @@
+resource "google_service_account" "object_encryptor_service_account" {
+  account_id   = "${var.name_prefix}-object-encryptor"
+  display_name = "${var.name_prefix} Object Encryptor DEV"
+}
+
+/*
+resource "google_service_account_iam_binding" "object_encryptor_iam" {
+  service_account_id = google_service_account.object_encryptor_service_account.name
+  role               = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+  members            = [
+    google_service_account.object_encryptor_service_account.member,
+  ]
+}
+*/
+
+resource "google_project_iam_binding" "object_encryptor_iam" {
```

Comment:
Nejde použít `google_service_account_iam_binding`, ale je potřeba `google_project_iam_binding`

https://keboolaglobal.slack.com/archives/C05H32UUJK1/p1692973302350969?thread_ts=1692971012.248309&cid=C05H32UUJK1

Ono je to napsaný v dokumentaci hned v prvním odstavci, ale tu větu jsem si musel přečíst 5x abych to pochopil
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_service_account_iam

Reply:
Precetl jsem tu dokumentaci 2x, ale chytrej z toho taky nejsem :smile: Budu ti verit.

Reply:
When managing IAM roles, you can treat a service account **either as a resource or as an identity**. This resource **(míněno google_service_account_iam_binding)** is to add iam policy bindings to a service account resource **(což není to co chcem, protože sice tu zakládám SA jako TF resource, ale chci ji použít jako identity)**, such as allowing the members to run operations as or modify the service account. To configure permissions for a service account on other GCP resources **(což je to co chcem, nastavit tomu SA přístup ke KMS)**, use the [google_project_iam](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam) set of resources.